src/main/java/com/kylenicholls/stash/parameterizedbuilds/rest/BuildResource.java

```diff
@@ -106,17 +115,31 @@ public Response triggerBuild(@Context final Repository repository, @PathParam("i
 	@GET
 	@Path(value = "getJobs")
 	public Response getJobs(@Context final Repository repository,
-			@QueryParam("branch") String branch, @QueryParam("commit") String commit, @QueryParam("prdestination") String prDestination) {
+			@QueryParam("branch") String branch, @QueryParam("commit") String commit, @QueryParam("prdestination") String prDestination, @QueryParam("prid") long prId) {
```

Comment:
in `/scripts/jenkins/pb-pr-trigger.js` line 22 you'll need to append the `prid` queryparam using the PR Id from `prJSON`